### PR TITLE
feat: update pod usage in flow

### PR DESCRIPTION
**Pull request by rutujasurve94**
Documentation for updating Pod usage in Flow
[Jira-Hub-273](https://jinaai.atlassian.net/browse/JINAHUBV1-273)

### CODE REVIEW

File: chapters/hub/guides/docker_pod/use-your-pod.md [hunk 1]
```diff
@@ -55,14 +55,15 @@ See `jina pod --help` for more usage.
 
 ## Use the Pod image via Flow API
 
-Finally, one can use it via Flow API as well, e.g.
+Pod image can be used in the Flow API as well. For instance:
```
Comment (alexcg1) [inline chapters/hub/guides/docker_pod/use-your-pod.md]: Change to `A Pod image can also be used in the Flow API as well:`

**Review by alexcg1 — CHANGES_REQUESTED** [review]

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:

**Comment by FionnD** [comment]
Did you find why it's showing a conflict @rutujasurve94? We don't have a folder called guides in hub.

**Comment by FionnD** [comment]
Hey @rutujasurve94 

Maybe just close this PR and open a new one? That might reset/fix the netlify issues?

**Comment by rutujasurve94** [comment]
I'm being able to merge it now

**Pull request merged**